app/build.gradle

```diff
@@ -373,9 +375,13 @@ android {
         if (backend == 'webkit' && !isWebKitAvailable())
             variant.setIgnore(true);
 
-        // MetaStore and AppLab only apply to oculusvr builds.
-        if ((store == 'metaStore' || store == "appLab") && !platform.startsWith('oculusvr'))
-            variant.setIgnore(true);
+        // MetaStore and AppLab only apply to oculusvr builds with gecko backend.
+        if ((store == 'metaStore' || store == "appLab")) {
+            if (!platform.startsWith('oculusvr'))
+                variant.setIgnore(true);
+            else
+                variant.setIgnore(backend != 'gecko')
```

Comment:
Aha! I thought that this change would be need because we didn't test chromium backend with store variants. Thanks for sharing context and this change is removed.